js/app.js

```diff
@@ -0,0 +1,180 @@
+// 1
+function setAlarm(employed, vacation) {
+    return employed && !vacation;
+}
+
+// 2
+function closeCompare(a, b, margin = 0) {
+    if (b - margin <= a && a <= b + margin) {
+        return 0;
+    }
+    if (a < b) {
+        return -1;
+    }
+    return 1;
+
+// 3
+    function or(a, b) {
+        return !(!a && !b);
+    }
+// true, false = true how the function works because a! is falsy and b! this would give false for !a && !b but we use !(...) so the outcome of what is inside the round brackets we invert it false is turned into truthy
+// true, true = true
+// false, true = true
+// false, false = false
+
+    function xor(a, b) {
```

Comment:
too complicated. Please look into this table:
<img width="552" alt="image" src="https://github.com/user-attachments/assets/5972d405-b3a4-42e5-b501-7fed5476003d">
When the returned value is not true? When the values are the same or when the values differ?

Reply:
As mentioned above, the exercise was not to use this logical operator ||.

Reply:
Still it can be done simpler. XOR returns true if the values are NOT THE SAME ;) You can use logical not and equality operator